### ISSUE TITLE
[WCAG] Guide Panel - Anchor links focus management

### ISSUE THREAD

**Issue by kenchase** [Canadian-Geospatial-Platform/GeoView#3533]
### Is there an existing issue for this?

- [x] I have searched the existing issues

### Current Behavior

Anchor sections use empty anchor elements, which break focus management. See the Footer Bar section of the guide for example.

In various scenarios (especially when or after the guide is viewed in full-screen mode), focus does not appear where it is expected to.

### Expected Behavior

When tabbing around the guide, focus indicators should be clearly visible and appear in a logical sequence.

### Steps To Reproduce
* Using Keyboard mode...
* Open the guide from the footerBar and select the "Footer Bar" section of the guide
* Tab and press the anchor links below "The Footer Bar menu bar has the following tabs:"
* Focus indicators are not implemented to show keyboard users where focus is
